### PR TITLE
add the lord of the rings API

### PR DESCRIPTION
**Pull request by gitfrosh**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [ ] Your additions are ordered alphabetically
- [ ] Your submission has a useful description
- [ ] The description does not end with punctuation
- [ ] Each table column should be padded with one space on either side
- [ ] You have searched the repository for any relevant issues or pull requests
- [ ] Any category you are creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -939,6 +939,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Ron Swanson Quotes](https://github.com/jamesseanwright/ron-swanson-quotes#ron-swanson-quotes-api) | Television | No | Yes | Unknown |
 | [STAPI](http://stapi.co) | Information on all things Star Trek | No | No | No |
 | [SWAPI](https://swapi.co) | Star Wars Information | No | Yes | Unknown |
+| [The one API](https://the-one-api.herokuapp.com/) | The Lord of the Rings API | `apiKey` | Yes | Unknown |
```
Comment (yannbertrand) [inline README.md]: Your title shouldn't end with API
```suggestion
| [The one](https://the-one-api.herokuapp.com/) | The Lord of the Rings API | `apiKey` | Yes | Unknown |
```

Comment (gitfrosh) [reply README.md]: I see, but it's the proper name of the API, and without the "API" in the end it makes no sense. As a compromise I'd suggest renaming it this way:

```suggestion
| [The Lord of the Rings](https://the-one-api.herokuapp.com/) | The Lord of the Rings API | `apiKey` | Yes | Unknown |
```

Comment (yannbertrand) [reply README.md]: 👌let's do this

Comment (gitfrosh) [reply README.md]: I'll commit :)

**Comment by yannbertrand** [comment]
Thank you!

**Pull request merged**